com.github.geigi.cozy.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "com.github.geigi.cozy",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.geigi.cozy",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-music",
```

Comment:
I meant to think about this before, but you might be able to expose this as :ro if you don't need to write anything back, which would improve the sandboxing.

Reply:
Oh I think I didn't understand the GitHub review process yet, or did you see my comment here already?
The user can add new audio files by drag & drop: Those will be copied to the selected directory and scanned by cozy. So I think we need rw access here?

Reply:
Yeah, lgtm